pkgs/by-name/ne/netfetch/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  stdenv,
+}:
+let
+  version = "5.2.10";
+in
+buildGoModule {
+  pname = "netfetch";
+  version = "5.2.10";
```

Comment:
I'd use either let or rec here, but writing out version twice doesn't make sense IMO

Reply:
Opted for rec version, i like it, less verbosity but i'm open to anything